src/components/DataController/DataController.js

```diff
@@ -31,12 +37,14 @@ function DataController({currentRegionBoundary}) {
         }
         return dataStrings;
       }
-  }  
+  }  */
 
   return (
     <div className="DataController">
         <br/>
         {monthlyTimeSeries ? <AnnualCycleGraph annualData={monthlyTimeSeries}/> : "No Data Available"}
+        <br/>
+        {longTermTimeSeries ? <LongTermAverageGraph longTermData={longTermTimeSeries}/> : "No Data Available"}
```

Comment:
Since we have two graphs now, could we edit the text here to be more specific and say which data was missing, instead of repeating "No Data Available" twice?